java/config/checkstyle/checkstyle.xml

```diff
@@ -308,13 +316,6 @@
       <message key="ws.notPreceded"
              value="GenericWhitespace ''{0}'' is not preceded with whitespace."/>
     </module>
-    <!-- Suppression for block code until we find a way to detect them properly
-         until https://github.com/checkstyle/checkstyle/issues/15769 -->
-    <module name="SuppressionXpathSingleFilter">
-      <property name="checks" value="Indentation"/>
-      <property name="query" value="//SLIST[not(parent::CASE_GROUP)]/SLIST
-                                   | //SLIST[not(parent::CASE_GROUP)]/SLIST/RCURLY"/>
-    </module>
```

Comment:
The issue mentioned in the comment has been fixed, so I guess it's safe to remove this suppression rule.